test/functionalTest/harnessFunctions.sh

```diff
@@ -378,6 +378,12 @@ function localBrokerStart()
   then
     port=$CB_PORT
     CB_START_CMD="$CB_START_CMD_PREFIX -port $CB_PORT  -pidpath $CB_PID_FILE  -dbhost $dbHost:$dbPort -db $CB_DB_NAME -dbPoolSize $POOL_SIZE -t $traceLevels $IPvOption $extraParams"
+  elif [ "$role" == "CBHA" ]
```

Comment:
To implement the test case for this PR (test/functionalTest/cases/4069_status_inactive_not_working_with_cache_in_ha/status_inactive_not_working_with_cache_in_ha.test) we have needed to extend the testing framework with a new role: CB for HA testing. It's like the usual CB but sharing the same database.

Thus, new HA cases can be tested from now on :)

NTC (informative)